--- a/Makefile.in
+++ b/Makefile.in
@@ -774,12 +774,13 @@
 	cp -r -t "$(distdir)" $(installFiles)
 
 install-exec-hook:
-	mkdir -p "$(prefix)/$(productID)"
-	$(INSTALL) $(installFiles) "$(prefix)/$(productID)"
+	mkdir -p "$(DESTDIR)/usr/games"
+	$(INSTALL) $(installFiles) "$(DESTDIR)/usr/games"
 
 install-data-local:
-	cp -r -t "$(prefix)/$(productID)" $(installData)
-	mkdir -p "$(prefix)/$(productID)/Songs"
+	mkdir -p "$(DESTDIR)/usr/share/stepmania"
+	cp -r -t "$(DESTDIR)/usr/share/stepmania" $(installData)
+	mkdir -p "$(DESTDIR)/usr/share/stepmania/Songs"
 
 uninstall-hook:
 	rm -f "$(prefix)/$(productID)/stepmania"
