--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -476,7 +476,7 @@
 ScoreDisplayPercentage.cpp ScoreDisplayPercentage.h ScoreDisplayRave.cpp ScoreDisplayRave.h \
 SongPosition.cpp SongPosition.h
 
-PCRE = ../extern/pcre/get.c ../extern/pcre/internal.h ../extern/pcre/maketables.c ../extern/pcre/pcre.c ../extern/pcre/pcre.h ../extern/pcre/study.c
+PCRE =
 EXTRA_DIST += ../extern/pcre/chartables.c
 
 Lua = ../extern/lua-5.1/src/lapi.c ../extern/lua-5.1/src/lauxlib.c ../extern/lua-5.1/src/lbaselib.c ../extern/lua-5.1/src/lcode.c ../extern/lua-5.1/src/ldblib.c \
--- a/src/RageUtil.cpp
+++ b/src/RageUtil.cpp
@@ -1342,7 +1342,7 @@
 	return true;
 }
 
-#include "../extern/pcre/pcre.h"
+#include <pcre.h>
 void Regex::Compile()
 {
 	const char *error;
--- a/configure.ac
+++ b/configure.ac
@@ -382,6 +382,8 @@
 dnl Add these after all tests are done as they don't exist just yet.
 LIBS="$LIBS $FFMPEG_LIBS"
 
+LIBS="$LIBS -lpcre"
+
 AC_CONFIG_FILES(Makefile)
 AC_CONFIG_FILES(bundle/Makefile)
 AC_CONFIG_FILES(src/Makefile)
